GVFS/GVFS.Virtualization/Projection/GitIndexProjection.cs

```diff
@@ -1172,7 +1177,7 @@ private void UpdatePlaceholders()
                             if (!isProjected ||
                                 !isFolder ||
                                 (!GVFSPlatform.Instance.KernelDriver.EnumerationExpandsDirectories &&
-                                !this.context.FileSystem.DirectoryExists(Path.Combine(this.context.Enlistment.WorkingDirectoryRoot, folderPlaceholder.Path))))
+                                folderPlaceholder.IsTombstoneFolder))

```

Comment:
If we decide to only record possible tombstones when the platform supports it, can/should we remove the `EnumerationExpandsDirectories` check?

Reply:
That should be fine since there shouldn't ever be any folder placeholder marked as possible tombstones.